.github/workflows/tests_primary.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [12]
+        node-version: [12, 16]
```

Comment:
Should we move half of these bots to secondary? I am afraid our primary is getting big again.